src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -335,6 +278,52 @@ public void OnHide()
         Hide();
     }
 
+    /// <summary>
+    ///   Searches the processes description string for compound name and input action
+    ///   "keys" and turns them into a BBCode string with matching values.
+    /// </summary>
+    private string ParseProcessesDescription()
+    {
+        if (string.IsNullOrEmpty(ProcessesDescription))
+            return string.Empty;
+
+        var inputEvents = Settings.GetCurrentlyAppliedControls();
+
+        var result = Regex.Replace(TranslationServer.Translate(ProcessesDescription), @"\[(.*?)\]", found =>
```

Comment:
I think this part (see one of my previous comments) would be worth it to generalize. Also there should be an escape option `\[stuff]` should be output as `[stuff]` and not attempted to be parsed.

Reply:
I've now moved the logic to separate class that inherits RichTextLabel so now it should be reusable for other places (just drag and drop the rich text label scene, script is already attached). Also added the mentioned escape option.